addons/event_booth/models/event_event.py

```diff
@@ -51,7 +51,7 @@ def _compute_event_booth_ids(self):
                         for attribute_name in self.env['event.type.booth']._get_event_booth_fields_whitelist()
                     }) for line in event.event_type_id.event_type_booth_ids
                 ]
-            event.event_booth_ids = command
+            event.sudo().event_booth_ids = command
```

Comment:
This one is strange. If you modify the event type, why would we need to sudo the update to booth ids ?